homeworks/01-bubble-sort/src/components/Pillar/Pillar.tsx

```diff
@@ -0,0 +1,16 @@
+import { Component } from "react";
+
+interface PillarProps {
+  array: number[];
+}
+interface PillarState {
+  arr: number[];
+}
+
+export class Pillar extends Component<PillarProps, PillarState> {
+  render() {
+    return this.props.array.map((el, index) => (
```

Comment:
What is `el`? Is it `element`? DOM Element? And why it goes to `height` if it's so?

It's unreadable naming. Do not shorten variable names.